released/discovery_center/mission_3239/provider.tf

```
@@ -0,0 +1,28 @@
+
+terraform {
+  required_providers {
+    btp = {
+      source  = "sap/btp"
+      version = ">=1.0.0"
```

I would recommend to use a bit more restrictive version

```suggestion
      version = "~> 1.3.0"
```